airflow/cli/commands/webserver_command.py

```diff
@@ -263,40 +264,31 @@ def monitor_gunicorn(gunicorn_master_proc):
                 sys.exit(gunicorn_master_proc.returncode)
 
         if args.daemon:
-            base, ext = os.path.splitext(pid)
-            ctx = daemon.DaemonContext(
-                pidfile=TimeoutPIDLockFile(base + "-monitor" + ext, -1),
-                files_preserve=[handle],
-                stdout=stdout,
-                stderr=stderr,
-                signal_map={
-                    signal.SIGINT: kill_proc,
-                    signal.SIGTERM: kill_proc
-                },
```

Comment:
I moved the signal handling to `monitor_gunicorn` function.

Reply:
Given these exist as properties of DaemonContext I would say we leave them there.

Reply:
I've moved those to `monitor_gunicorn` to avoid code duplication (we register them in both daemon and not versions).